front/src/modules/ui/table/components/EntityTableHeaderOptions.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { DropdownButton } from '@/ui/dropdown/components/DropdownButton';
+import { StyledDropdownMenu } from '@/ui/dropdown/components/StyledDropdownMenu';
+import { StyledDropdownMenuItemsContainer } from '@/ui/dropdown/components/StyledDropdownMenuItemsContainer';
+import { useDropdownButton } from '@/ui/dropdown/hooks/useDropdownButton';
+import { ViewFieldMetadata } from '@/ui/editable-field/types/ViewField';
+import {
+  IconArrowNarrowLeft,
+  IconArrowNarrowRight,
+  IconEyeOff,
+} from '@/ui/icon';
+import { MenuItem } from '@/ui/menu-item/components/MenuItem';
+
+import { ColumnHeaderDropdownId } from '../constants/ColumnHeaderDropdownId';
+import { useTableColumns } from '../hooks/useTableColumns';
+import { ColumnDefinition } from '../types/ColumnDefinition';
+
+const StyledDropdownContainer = styled.div`
+  left: 0px;
+  position: absolute;
+  top: 32px;
+  z-index: 1;
+`;
+type OwnProps = {
+  column: ColumnDefinition<ViewFieldMetadata>;
+  isFirstColumn: boolean;
+  isLastColumn: boolean;
+};
+export const EntityTableHeaderOptions = ({
+  column,
+  isFirstColumn,
+  isLastColumn,
+}: OwnProps) => {
+  const theme = useTheme();
+
+  const {
+    handleColumnVisibilityChange,
+    handleColumnLeftMove,
+    handleColumnRightMove,
+  } = useTableColumns();
+
+  const { closeDropdownButton } = useDropdownButton({
+    dropdownId: ColumnHeaderDropdownId,
+  });
+
+  const handleColumnMoveLeft = () => {
+    closeDropdownButton();
+    if (isFirstColumn) return;
+    else handleColumnLeftMove(column);
+  };
+
+  const handleColumnMoveRight = () => {
+    closeDropdownButton();
+    if (isLastColumn) return;
+    else handleColumnRightMove(column);
+  };
+
+  const handleColumnVisibility = () => {
+    handleColumnVisibilityChange(column);
+  };
+
+  return (
+    <DropdownButton
+      dropdownId={ColumnHeaderDropdownId}
+      dropdownComponents={
+        <StyledDropdownContainer>
```

Comment:
you should not need this anymore!

Reply:
If I remove this, it's messing up the styling

https://github.com/twentyhq/twenty/assets/13139771/d919ea4b-b334-49c8-b0b6-f584961f09e1